fhir-ig-mcode/src/test/java/com/ibm/fhir/ig/mcode/test/ConstraintGeneratorTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.mcode.test;
+
+import com.ibm.fhir.ig.mcode.MCODEResourceProvider;
+import com.ibm.fhir.model.annotation.Constraint;
+import com.ibm.fhir.model.resource.StructureDefinition;
+import com.ibm.fhir.model.type.Extension;
+import com.ibm.fhir.model.util.ModelSupport;
+import com.ibm.fhir.profile.ProfileSupport;
+import com.ibm.fhir.registry.resource.FHIRRegistryResource;
+import com.ibm.fhir.registry.spi.FHIRRegistryResourceProvider;
+
+public class ConstraintGeneratorTest {
+    public static void main(String[] args) {
```

Comment:
should it be a proper test?

Reply:
This is more of a diagnostic driver to print out the generated constraints for manual inspection. I don't anticipate that we will make this part of the automated unit tests for this module.